doozer/doozerlib/cli/scan_osh.py

```diff
@@ -363,9 +363,13 @@ def get_packages_with_scan_issues(self, mapping: dict):
                 self.runtime.logger.info(f"No successful scan found for package {brew_package_name}")
                 continue
 
-            if self.check_if_scan_issues_exist(task_id=data["latest_coverity_scan"],
-                                               nvr=data["latest_coverity_scan_nvr"]):
-                nvrs_with_scan_issues[brew_package_name] = data
+            try:
+                if self.check_if_scan_issues_exist(task_id=data["latest_coverity_scan"],
+                                                   nvr=data["latest_coverity_scan_nvr"]):
+                    nvrs_with_scan_issues[brew_package_name] = data
+            except Exception:
+                self.runtime.logger.warning(f"Don't see scan-results-imp.js. Skipping {brew_package_name}")
```

Comment:
think this is an anti-pattern - including internal implementation details of a func outside of it

Reply:
Ahh! Noted! Thank you!

Hope https://github.com/openshift-eng/art-tools/pull/198/commits/292176b0bfab66bb36c0f6d4252c48d2a2470a25 addresses that